docs/source/github_api_access.rst

```diff
@@ -0,0 +1,12 @@
+.. _github_api_access:
```

Comment:
Nit: I'm getting some warning when running `tox -e docs` locally:

```
docs/source/github_api_access.rst:1: WARNING: Duplicate explicit target name: "github_api_access".
```

Does it matter?

Reply:
Yeah, i had to add `github_api_access.rst` to `exclude_patterns`